packages/website/src/pages/foundations/Iconography.tsx

```diff
@@ -83,11 +83,10 @@ const IconsListLayout = <T,>({table}: RowLayoutProps<T>) => {
     );
 };
 
-const CardLayout: TableLayout = {
-    name: 'Cards',
-    Body: IconsListLayout,
-    Header: () => null,
-};
+const CardLayout: TableLayout = ({children}) => <>{children}</>;
+CardLayout.displayName = 'Cards';
+CardLayout.Body = IconsListLayout;
+CardLayout.Header = () => null;
```

Comment:
Somehow the table header background color has changed (compared to master)

Master 👇 
![image](https://github.com/coveo/plasma/assets/35579930/2f340151-5837-4baa-bc2e-dd6c0c973bbc)


This branch 👇 
![image](https://github.com/coveo/plasma/assets/35579930/4d374981-5a99-43d9-b3be-64bfe86ee79b)

Reply:
Fixed